addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3178,6 +3194,14 @@ registry.SnippetSave = SnippetOptionWidget.extend({
             dialog.on('closed', this, () => resolve());
         });
     },
+    async _confirmReload() {
+        return new Promise(resolve => {
+            Dialog.confirm(this, _t("To save a snippet, we need save all your changes to the current page and reload it, are you sure you want to proceed?"), {
```

Comment:
"we need to" ?

Reply:
What do you think of:
"To create a custom snippet, save your changes and reload the page."
Btn: "Save and reload" "Discard"
?

Reply:
Yes, better, like in master for theme options
cc @sdegueldre